src/Umbraco.Core/Services/CacheInstructionServiceProcessInstructionsResult.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+
+namespace Umbraco.Cms.Core.Services
+{
+    /// <summary>
+    /// Defines a result object for the <see cref="ICacheInstructionService.ProcessInstructions(bool, DateTime)"/> operation.
+    /// </summary>
+    public class CacheInstructionServiceProcessInstructionsResult
+    {
+        private CacheInstructionServiceProcessInstructionsResult()
+        {
+        }
+
+        public int NumberOfInstructionsProcessed { get; private set; }
```

Comment:
Same thing here, if these are readonly they should just be { get; }

Reply:
This one does need to be `private set` actually as it's set not in the constructor but in the static factory methods.